sdks/rust/Cargo.toml

```diff
@@ -14,7 +14,7 @@
 
 [package]
 name = "agones"
-version = "1.34.0"
+version = "1.35.0"
```

Comment:
I'm curious why this version lags all of the other ones. Should this be set to 1.35.0 for the release and then bumped to 1.36.0-dev after the release like the rest of the version variables?

Reply:
This change is to post the release version 1.35.0 on [Crates.io](https://crates.io/crates/agones). Will update it to 1.36.0-dev👍🏻